SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialStatics.cpp

```diff
@@ -107,8 +107,20 @@ bool USpatialStatics::IsSpatialMultiWorkerEnabled(const UObject* WorldContextObj
 		return SpatialGDKSettings->bOverrideMultiWorker.GetValue();
```

Comment:
If the user overrides the boolean option, the soft class path option is ignored. I think we should invert these checks